e2e-tests/stream-mode-tests.sh

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env bash
+
+print_prefix="E2E: STREAM MODE TEST:"
+echo "${print_prefix} START OF STREAM MODE TEST"
+echo "${print_prefix} STARTING OPENMRS SERVER."
+docker-compose -f openmrs-compose.yaml up -d --remove-orphans
+sleep 2m;
+openmrs_start_wait_time=0
+contenttype=$(curl -o /dev/null --head -w "%{content_type}\n" -X GET -u admin:Admin123 --connect-timeout 5 \
+ --max-time 20 http://localhost:8099/openmrs/ws/rest/v1/visittype 2>/dev/null | cut -d ";" -f 1)
+ until [[ ${contenttype} == "application/json" ]]; do
```

Comment:
Please factor out the common pieces between this script and the batch mode. Move those common parts into bash functions and call those functions in both cases (instead of copying).
Please also take a look at the refactorings in PR #142 .